cdoc/job-tutorial.md

```diff
@@ -0,0 +1,268 @@
+<!--
```

Comment:
It's better to rename this file to `README.zh.md` and put under `job-tutorial` dir instead of create a `cdoc` dir.

Reply:
OK